src/routes/token/index.ts

```diff
@@ -1,9 +1,11 @@
 import express from 'express';
 
 import addTokenRoute from './addToken';
+import mintToken from './mintToken';
 
 const router = express.Router();
 
 router.get('/add', addTokenRoute);
```

Comment:
change the method to POST and remove the add in the route. POST /tokens

add a middleware where you check the headers for a field called 'Authorization' that should match the `process.env.ADMIN_KEY`

next()